config/setup.sql

```diff
@@ -6,7 +6,9 @@ CREATE TABLE IF NOT EXISTS users(
 		id INT(11) PRIMARY KEY NOT NULL AUTO_INCREMENT,
 		username VARCHAR(15) NOT NULL,
 		mail VARCHAR(30) NOT NULL,
-		password VARCHAR(150) NOT NULL
+		password VARCHAR(150) NOT NULL,
+		notifications BOOLEAN NOT NULL DEFAULT true,
```

Comment:
Notifications should not be in true, in case of a user who wants them, we can put in to true. Beside they must be set as false. 
```suggestion
		notifications BOOLEAN NOT NULL DEFAULT true,
```

Reply:
lis le sujet batard